app/controllers/course/assessment/submission/submissions_controller.rb

```diff
@@ -7,10 +7,16 @@ class Course::Assessment::Submission::SubmissionsController < \
   skip_authorize_resource :submission, only: [:edit, :update, :auto_grade]
   before_action :authorize_submission!, only: [:edit, :update]
   before_action :load_or_create_answers, only: [:edit, :update]
+  before_action :load_course_students, only: [:index]
```

Comment:
Can we move this to the action, that is load the course_users in index action, use a `before_action` here looks a bit weird to me. 

Normally we use `before_action` if there is authorisation ( we don't want certain action to be executed if user is unauthorised ) or the resources are shared between multiple actions.